### PR TITLE
[Task8] Added API testing

### PR DESCRIPTION
**Pull request by Zhukova-Iryna**

### CODE REVIEW

File: src/main/java/rest_api/petshop/Category.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    int id;
+    String name;
+
+    public int getId() {
+        return id;
+    }
+
+    public Category setId(int id) {
+        this.id = id;
+        return new Category();
```
Comment (DmitriyBondarenko) [inline src/main/java/rest_api/petshop/Category.java]: `return this;`?

File: src/main/java/rest_api/petshop/OrderStatus.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package rest_api.petshop;
+
+public enum OrderStatus {
+    placed, approved, delivered
```
Comment (DmitriyBondarenko) [inline src/main/java/rest_api/petshop/OrderStatus.java]: Please, follow the Java Code Convention and keep the Enums naming format

File: src/main/java/rest_api/petshop/PetStatus.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package rest_api.petshop;
+
+public enum PetStatus {
+    available, pending, sold
```
Comment (DmitriyBondarenko) [inline src/main/java/rest_api/petshop/PetStatus.java]: The same here

File: src/test/java/rest_api/petshop_test/PetTest.java [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+public class PetTest {
+    private final static String BASE_URI = "https://petstore.swagger.io/v2";
+    private RequestSpecification requestSpecification;
+    private PetDto pet;
+    private UserDto user;
+    private OrderDto order;
+
+    @BeforeClass
+    public void configureRestAssure() {
```
Comment (DmitriyBondarenko) [inline src/test/java/rest_api/petshop_test/PetTest.java]: Please, move all this configuration to some `TestBase `class

And create some `unconfigureRestAssured()` method where all settings should be reset.

File: src/main/java/rest_api/petshop/dto/OrderDto.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        this.complete = complete;
+        return this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        OrderDto orderDto = (OrderDto) o;
+        return getId() == orderDto.getId() &&
```
Comment (DmitriyBondarenko) [inline src/main/java/rest_api/petshop/dto/OrderDto.java:79]: Not sure that we need all this fields in equals overriding. Just use ID here and in all other DTOs

File: src/main/java/rest_api/petshop/Category.java (hunk 1 above)
Comment (Zhukova-Iryna) [reply src/main/java/rest_api/petshop/Category.java]: Done

File: src/main/java/rest_api/petshop/OrderStatus.java (hunk 2 above)
Comment (Zhukova-Iryna) [reply src/main/java/rest_api/petshop/OrderStatus.java]: Done

File: src/main/java/rest_api/petshop/PetStatus.java (hunk 3 above)
Comment (Zhukova-Iryna) [reply src/main/java/rest_api/petshop/PetStatus.java]: Done

File: src/test/java/rest_api/petshop_test/PetTest.java (hunk 4 above)
Comment (Zhukova-Iryna) [reply src/test/java/rest_api/petshop_test/PetTest.java]: Done

File: src/main/java/rest_api/petshop/dto/OrderDto.java (hunk 5 above)
Comment (Zhukova-Iryna) [reply src/main/java/rest_api/petshop/dto/OrderDto.java:79]: Done

**Review by DmitriyBondarenko — APPROVED** [review]

**Pull request merged**